Breeze2

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+# isort: skip
+import os
+import sys
+
+# Python <3.4 does not have pathlib
```

Comment:
Comment doesn't line up with code :)

Reply:
Oh yeah. That was a mentl shortcut. The comment should be:

| We do the check here, before importing pathlib becasue Python <3.4 does not have pathlib and import would fail with cryptic "missing pathlib module" error.